src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -192,5 +192,17 @@ public static BaselineState CreateBaselineState(BaselineStateVersionOne v1Baseli
                     return BaselineState.None;
             }
         }
+        public static CodeFlowLocationImportance CreateCodeFlowLocationImportance(AnnotatedCodeLocationImportanceVersionOne v1AnnotatedCodeLocationImportance)
+        {
+            switch (v1AnnotatedCodeLocationImportance)
+            {
+                case AnnotatedCodeLocationImportanceVersionOne.Essential:
+                    return CodeFlowLocationImportance.Essential;
+                case AnnotatedCodeLocationImportanceVersionOne.Important:
+                    return CodeFlowLocationImportance.Important;
+                default:
+                    return CodeFlowLocationImportance.Unimportant;
```

Comment:
The default for v1's `annotatedCodeLocation.importance` property was `important`.

(The same is true in v2, but that's neither here nor there, since you're processing a v1 value here.)

Reply:
Fixed